bin/init.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+
+var filesToCopy = [
+  '.travis.yml',
+  'gulpfile.js'
+];
+
+function copyFileSync(srcFile, destFile, encoding) {
```

Comment:
I don't see `sendfile()` or the like in node's `fs` module, so I just did what was fast/easy/dumb. It's likely not the best way but then I don't see node abstracting out system calls so I'm not sure.

A quick google finds a lot of people copying files this way :unamused: